<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control_properties</class>
 <widget class="QWidget" name="Control_properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 241, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>按钮命令数据：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>按钮命名：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BTN_name">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>按钮宽度：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BTN_hight">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>按钮宽度：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BTN_width">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>50</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BTN_Set">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>219</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{color: rgb(255, 255, 255); background-color: rgb(20, 70, 130);border:none;border-radius:3px;padding:2px 4px;font: 75 11pt &quot;微软雅黑&quot;;}
QPushButton:hover{ color: rgb(255, 255, 255);border:none;border-radius:5px;padding:2px 4px;background-color: rgb(190, 190, 190);font: 75 11pt &quot;微软雅黑&quot;;}



</string>
       </property>
       <property name="text">
        <string>更改属性</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>竖向坐标Y：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>横向坐标X：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BTN_Y">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>80</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="BTN_X">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="BTN_DATAS">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>301</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QPushButton" name="BTN_DELETE">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{color: rgb(255, 255, 255); background-color: rgb(20, 70, 130);border:none;border-radius:3px;padding:2px 4px;font: 75 11pt &quot;微软雅黑&quot;;}
QPushButton:hover{ color: rgb(255, 255, 255);border:none;border-radius:5px;padding:2px 4px;background-color: rgb(190, 190, 190);font: 75 11pt &quot;微软雅黑&quot;;}



</string>
       </property>
       <property name="text">
        <string>删除该控件</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
